autonomy/cli/scaffold_fsm.py

```diff
@@ -191,24 +192,29 @@ def author(self) -> str:
         return self.ctx.agent_config.author
 
     @property
-    def rounds(self) -> Set[str]:
+    def all_rounds(self) -> Set[str]:
         """Rounds"""
         return self.dfa.states
 
     @property
-    def non_degenerate_rounds(self) -> Set[str]:
+    def rounds(self) -> Set[str]:
```

Comment:
addresses [comment](https://github.com/valory-xyz/open-autonomy/pull/1297#discussion_r964929869) in #1297